libs/libc/signal/sig_ismember.c

```diff
@@ -63,17 +66,53 @@
  *
  ****************************************************************************/
 
-int sigismember(FAR const sigset_t *set, int signo)
+int nxsig_ismember(FAR const sigset_t *set, int signo)
 {
-  int ret = ERROR;
-
   /* Verify the signal */
 
-  if (GOOD_SIGNO(signo))
+  if (!GOOD_SIGNO(signo))
+    {
+      return ERROR;
```

Comment:
```suggestion
      return -EINVAL;
```
There are probably Return Value comment above that need to be made to match too.

Reply:
Done.